### ISSUE TITLE
[Minor] Release email URLs not clickable in some email clients

### ISSUE THREAD

**Issue by stefanvodita** [apache/lucene#12119]
### Description

The release email includes links [for download](https://github.com/apache/lucene/blob/main/dev-tools/scripts/releaseWizard.yaml#L95) and [to the change log](https://github.com/apache/lucene/blob/main/dev-tools/scripts/releaseWizard.yaml#L104). These URLs are post-fixed with `>`. Some email clients correctly recognise that this is not part of the URL (e.g. Gmail), while others do not (e.g. Outlook).

It's a bad experience for users to get a 404 when checking out a new release. Maybe we can remove the `< ... >` around the URLs?

**Comment by stefanvodita** [comment]
I've not observed the same thing with the new 9.12.2 and 10.2.2 releases. Maybe it was just a one-time issue. I'm not convinced my original understanding was right anyway.

**Comment by rmuir** [comment]
Maybe it has to do with whether the email had html or not, I could see the `>` becoming a `&gt;` and then get treated differently by different clients?

**Comment by stefanvodita** [comment]
Yeah, maybe. Both Gmail and Outlook work correctly for me now though 🤷
